server/lib/pencil_space_server_web/controllers/game_controller.ex

```diff
@@ -1,36 +1,55 @@
 defmodule PencilSpaceServerWeb.GameController do
   @moduledoc """
-  Game controller that deals with creating/joining games over HTTP.
+  Creating/joining games over HTTP.
   """
 
   use PencilSpaceServerWeb, :controller
-  alias PencilSpaceServer.{GameMonitor}
+  alias PencilSpaceServer.Game
+  import PencilSpaceServerWeb.ControllerHelpers
```

Comment:
MINOR: You can also specify the functions that are being imported from this module. 

https://elixir-lang.org/getting-started/alias-require-and-import.html#import